view_component.gemspec

```
@@ -40,7 +40,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "haml", "~> 6"
   spec.add_development_dependency "jbuilder", "~> 2"
   spec.add_development_dependency "m", "~> 1"
-  spec.add_development_dependency "minitest", "= 5.6.0"
+  spec.add_development_dependency "minitest", "= 5.18.0"
```

```suggestion
  spec.add_development_dependency "minitest", "~> 5.18"
```
 This may be more appropriate?

---

[This](https://github.com/ViewComponent/view_component/commit/4db8e52310226642f132de8a384d3dc789aad636) is the commit that introduced the `=` notation.